### DISCUSSION TITLE
Что означает пункт "Видимость mp3" у плейлиста?

### DISCUSSION THREAD

**Discussion by AlexByte** [Chimildic/YaMuTools#80] [Вопрос/ответ] [answered]
> Новый инструмент - видимость mp3. Позволяет опубликовать или скрыть ваши mp3-файлы, которые находятся в плейлисте.

Куда эти mp3 публикуются? mp3 файлы, вручную загруженные, в любом случае приватные же

**Comment by kisoid** [comment, marked as answer]
https://github.com/Chimildic/YaMuTools/discussions/73
